tests/integration/conf.py

```diff
@@ -0,0 +1,83 @@
+"""Constants and variables used in integration tests."""
+import logging
+import os
+from pathlib import Path
+
+import aiohttp
+
+FORMAT = "[%(asctime)s][%(name)s][%(process)d %(processName)s][%(levelname)-8s](L:%(lineno)s) %(funcName)s: %(message)s"
+DATE_FORMAT = "%Y-%m-%d %H:%M:%S"
+logging.basicConfig(format=FORMAT, datefmt=DATE_FORMAT)
+
+testfiles_root = Path(__file__).parent.parent / "test_files"
+test_xml_files = [
+    ("study", "SRP000539.xml"),
+    ("sample", "SRS001433.xml"),
+    ("run", "ERR000076.xml"),
+    ("experiment", "ERX000119.xml"),
+    ("experiment", "paired.xml"),
+    ("experiment", "sample_description.xml"),
+    ("analysis", "ERZ266973.xml"),
+    ("analysis", "processed_reads_analysis.xml"),
+    ("analysis", "reference_alignment_analysis.xml"),
+    ("analysis", "reference_sequence_analysis.xml"),
+    ("analysis", "sequence_assembly_analysis.xml"),
+    ("analysis", "sequence_variation_analysis.xml"),
+    ("dac", "dac.xml"),
+    ("policy", "policy.xml"),
+    ("dataset", "dataset.xml"),
+    ("image", "images_single.xml"),
+    ("bpdataset", "template_dataset.xml"),
+]
+test_json_files = [
+    ("study", "SRP000539.json", "SRP000539.json"),
+    ("sample", "SRS001433.json", "SRS001433.json"),
+    ("dataset", "dataset.json", "dataset.json"),
+    ("run", "ERR000076.json", "ERR000076.json"),
+    ("experiment", "ERX000119.json", "ERX000119.json"),
+    ("analysis", "ERZ266973.json", "ERZ266973.json"),
+]
+test_schemas = [
+    ("submission", 200),
+    ("study", 200),
+    # ("project", 200),
```

Comment:
not sure we ever had a use case for project, we didn't eve do a JSON schema, we might want o deprecate it :man_shrugging:

Reply:
I don't think we would, but project is one of the DB schemas, so I do think it's nice to keep it visible.

Reply:
I was talking about `ENA.project.xsd` but I assume you mean the User projects :)